### DISCUSSION TITLE
realtime-py only works with secret key?

### DISCUSSION THREAD

**Discussion by qishansun** [supabase/supabase#9663] [Questions]
Thanks for  building realtime-py package. 

Is it possible to use ANON key for supabase change notificaiton from client side? 
Secret key works as expected. But if  changed to ANON key, i will get websock 401 unauthorized error.

**Comment by w3b6x9** [comment]
@qishansun what do you mean by secret key? is it your `service_role` key?

**Reply by qishansun** [reply]
yes, service_role secret key.
